hasher-matcher-actioner/hmalib/storage/hashstore.py

```diff
@@ -0,0 +1,16 @@
+from mypy_boto3_dynamodb.service_resource import Table
+
+class _IDynamoDBItem():
+    """ Clowny type as an interfaces. True interfaces in python are nightmares.
+    """
+    def to_dynamodb_item(self) -> dict:
+        pass
+
+class HashStore:
```

Comment:
blocking: I don't think this is a HashStore. I think this is the MatchStore, MatchRecords, PDQMatches, or some variation.

@BarrettOlson , thoughts?